components/bl-badge-component/src/content.js

```diff
@@ -0,0 +1,20 @@
+import { Icon } from './content/icon';
```

Comment:
Create an index with reexporting all types of content and here will be

import { Icon, Image, Text } from './content';